daemon/daemon_main.go

```diff
@@ -104,6 +104,14 @@ var (
 )
 
 func init() {
+	// Reset the help function to also exit, as we block elsewhere in interrupts
+	// and would not exit when called with -h.
+	oldHelpFunc := RootCmd.HelpFunc()
+	RootCmd.SetHelpFunc(func(c *cobra.Command, a []string) {
```

Comment:
There are two `init` functions in this module. This one, before your change, only initialized logging. There is a second one which initializes commands and flags:

https://github.com/cilium/cilium/blob/master/daemon/daemon_main.go#L282

That's probably a bit better place to do that. Eventually somewhere here:

https://github.com/cilium/cilium/blob/master/daemon/daemon_main.go#L119

Reply:
Oh, very good catch! I have moved it. A+1 :)